### PR TITLE
Simplifying folder names

### PR DESCRIPTION
**Pull request by slewis74**
This PR removes the Flavour name from the folder and csproj names.

Experience from working on the extension libraries with long and extension specific names was that it made things harder to read, and harder to copy when you wanted to create something new. Browsing around the solution is simpler when you only have to look for the `Server` folder. Similarly here the folders have been simplified to `Calamari` and `Sashimi`.

Review is possibly easiest by pulling the branch and opening in the IDE

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -97,10 +97,10 @@ Task("Test")
 Task("PublishCalamariProjects")
    .IsDependentOn("Build")
     .Does(() => {
-        var projects = GetFiles("./source/**/Calamari.*.csproj");
+        var projects = GetFiles("./source/**/Calamari.csproj");
 		foreach(var project in projects)
         {
-            var calamariFlavour = project.GetFilenameWithoutExtension().ToString();
+            var calamariFlavour = XmlPeek(project, "Project/PropertyGroup/AssemblyName");
```
Comment (slewis74) [inline build.cake:104]: the assembly name will always be the flavour name, even though the csproj has now been renamed

File: build.cake [hunk 2]
```diff
@@ -97,10 +97,10 @@ Task("Test")
 Task("PublishCalamariProjects")
    .IsDependentOn("Build")
     .Does(() => {
-        var projects = GetFiles("./source/**/Calamari.*.csproj");
+        var projects = GetFiles("./source/**/Calamari.csproj");
```
Comment (flin-8) [inline build.cake]: If this is going to be the template repo, we will need a new function to zip the Calamari test project

Comment (flin-8) [reply build.cake]: Actually I think we should have a template repository in Github.

The 🍰  files should all be consistent though

Comment (slewis74) [reply build.cake]: This is actually something I really like about this approach, and the adjustments that have been made in these cake files versus the ones we've got in the extension repos. The ps1 files are same in both cases but the cake files here are much cleaner, no mention of the repo specifics which means would could auto flow the same file to all repos and it'll work nicely.

**Review by flin-8 — APPROVED** [review]
Looking good!

**Pull request merged**